terraform/modules/networking/site-to-site-vpn/main.tf

```diff
@@ -0,0 +1,108 @@
+terraform {
+  required_version = ">= 1.6.6"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.54"
+    }
+  }
+}
+
+# VPN
+
+resource "aws_customer_gateway" "customer_gateway" {
+  type       = "ipsec.1"
+  bgp_asn    = "65000"
+  ip_address = var.customer_gateway_ip
+
+  tags = {
+    Name = "integrated-data-${var.vpn_name}-customer-gateway-${var.environment}"
+  }
+}
+
+resource "aws_vpn_gateway" "vgw" {
```

Comment:
Only comment would be if we re-use the vpn gateway when adding new vpn connections this resource might be better placed in the VPC module. But it's a bit of a premature optimisation since we're only really doing this for stagecoach.